_build/Extension-SetIdentityAndVersion.ps1

```diff
@@ -0,0 +1,198 @@
+# THIS SCRITP WILL NEED TO BE SPLITED NEXT TIME, GETTING TO LONG.
```

Comment:
It would be really useful to see an example (or multiple examples if appropriate) of a call to this file in a comment at the top. This will remove the need to decipher or reverse engineer what might be passed in.
Comments at the top which also explain what this file is and what it is used for can also be helpful for anyone looking at this in the future.